### ISSUE TITLE
Make a Buoy setup guide for community admins who don't have WP sites already

### ISSUE THREAD

**Issue by fabacab** [betterangels/buoy#65]
The biggest question we get is "where do I get Buoy," because people are used to Facebook and don't understand that the centralized model is method of social control that must be resisted. The real solution to this problem is to build community capacity in such a way as to ensure fully community-owned and operated infrastructure from the ground up, including the physical layer (cabling) that makes up telecom systems. However, that is beyond the scope of what most people are currently capable of, and the whole point of the Buoy project is that it aims bootstrap the awareness of this importance in non-technical people, so we need to start elsewhere.

One thing missing from our docs is a guide for how a non-technical person can set up their own Buoy instance. This is missing because there are so many ways to do it and our focus has recently been on organizations who already have WordPress websites up and running (for which the process of creating a Buoy is the same as installing any other plugin). With that knowledge finally common place (or common place enough, anyway, _sigh_), the next step is to create a guide for people who do not already have a running WordPress website but who want to make a Buoy for their community.

Such a guide MUST:
- focus and prioritize FSF-approved free software
- clearly describe any financial costs associated with a particular step (such as domain name registration)

Basically, this should be a technical "recipe," describing ingredients and yields, not a user manual on how WordPress works or how to install it somewhere in specific; that work has already been done and is available with any basic Internet search.

**Comment by fabacab** [comment]
One resource worth looking into for this is [SandStorm.io](https://SandStorm.io/), which provides free WordPress server hosting that is portable. The idea would be:
1. Create a Buoy on SandStorm Oasis's free tier (i.e., install WordPress, then install the Buoy plugin)
2. When you need more storage, either upgrade to the $9/mo "Basic" tier or download and install SandStorm on your own VM, export the WordPress app instance ("grain") from the Oasis hosted service and import it into your own SandStorm instance.
   - Your VM could be cloud-hosted (DigitalOcean/Linode/etc.) or running in your own house/office, although non-technical people will probably take the former route.

**Comment by fabacab** [comment]
Unfortunately, [Sandstorm uses a forked WordPress](https://github.com/dwrensha/wordpress-sandstorm) and its latest version is 4.3.2-alpha, but Buoy requires WordPress 4.4 or later due to our use of its CSPRNG polyfill (the `random_compat` library). This means Buoy will not run on Sandstorm, at least not yet.

**Comment by fabacab** [comment]
Another possibility that might be more compatible is using [YunoHost](https://yunohost.org/), a Debian-based Linux distro designed to be a Single Sign-On server solution. Might be less "user friendly" than Sandstorm, but worth having a look.

/cc @abiggerhammer

**Comment by fabacab** [comment]
Did some work on this by creating a [Installing Buoy on Sandstorm Oasis](/betterangels/buoy/wiki/Installing-Buoy-on-Sandstorm-Oasis) page on our wiki, but this is more of a guide for other writers since Sandstorm still isn't a usable platform for Buoy (at least until dwrensha/wordpress-sandstorm#12 is resolved).

Anyway, that's a start towards getting some documentation about this up. More would be better!

**Comment by hurtstotouchfire** [comment]
FWIW, I'm interested in considering this use-case one which we could address with back-end expansion.
